.github/dependabot.yml

```diff
@@ -16,7 +16,7 @@ updates:
   - package-ecosystem: "npm"
     directory: "/frontend"
     schedule:
-      interval: "weekly"
```

Comment:
Security badge is a phone icon instead of a lock pad. 

Before:
![Screenshot from 2022-05-03 12-59-19](https://user-images.githubusercontent.com/6226732/166502787-ceaef73d-e346-4554-a233-81ff89fdbf0c.png)

After:
![Screenshot from 2022-05-03 12-59-55](https://user-images.githubusercontent.com/6226732/166502806-de8745c9-781b-48ce-82ec-61208736903b.png)

Reply:
Weird image on "about page"

Before:
![Screenshot from 2022-05-03 13-22-10](https://user-images.githubusercontent.com/6226732/166505903-d8a55383-1b0b-474b-92e6-fef5c8d1bec8.png)

After:
![Screenshot from 2022-05-03 13-06-29](https://user-images.githubusercontent.com/6226732/166503939-caae5914-89e8-45dd-896e-87e114111ebc.png)